terraform/modules/__tests__/main-test.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) HashiCorp, Inc
+// SPDX-License-Identifier: MPL-2.0
+import "cdktf/lib/testing/adapters/jest"; // Load types for expect matchers
+import { TerraformStack, Testing } from "cdktf";
+import { HappyDNS } from "../main"
+import { Route53Record } from "@cdktf/provider-aws/lib/route53-record";
+import { AwsProvider } from "@cdktf/provider-aws/lib/provider";
+describe("happy DNS", () => {
+  it("creates a AWS Route53 record", () => {
+    const construct = Testing.synthScope((scope) => {
+      new HappyDNS(scope, "my-dns", "albName", "albZoneId", "zone.czi.technology", "zoneId", "stackName", "appName");
+    })
+    expect(construct).toHaveResourceWithProperties(Route53Record, {
+      type: "A",
+      name: "stackName-appName.zone.czi.technology",
+      zone_id: "zoneId", // weird, it changes from camelCase to snake_case otherwise the test will fail
```

Comment:
I think this is because node/TS standard is to use camelcase and TF standard is snakecase